modules/release-notes/con-relnotes-fixed-issues.adoc

```diff
@@ -22,17 +23,19 @@ In earlier versions of {product}, defining conditional policies with a policy ac
 
 This issue led to difficulties in defining conditional policies, hindering the configuration of permissions within the application.
 
-With this fix, you can now define conditional policies in {product-short} with permission policy actions of 'use'. link:{LinkRHIDPIssue}RHIDP-2237[(RHIDP-2237)]
+With this fix, you can now define conditional policies in {product-short} with permission policy actions of 'use'. 
+//link:{LinkRHIDPIssue}RHIDP-2237[RHIDP-2237]
 --
 
 Setting a custom route not working when using the {product-short} Operator on OpenShift::
 +
 --
 In earlier versions of the {product} Operator, setting a custom Route host on OpenShift Container Platform by using the `spec.application.route.host` field in the Custom Resource was not possible.
 
-    This limitation prevented users from configuring custom Route hosts, limiting their ability to customize the deployment environment.
+This limitation prevented users from configuring custom Route hosts, limiting their ability to customize the deployment environment.
 
-With this release, you can now set a custom Route host on OpenShift Container Platform by using the specified field in the Custom Resource. link:{LinkRHIDPIssue}RHIDP-2325[(RHIDP-2325)]
+With this release, you can now set a custom Route host on OpenShift Container Platform by using the specified field in the Custom Resource. 
```

Comment:
```suggestion
With this release, you can now set a custom route host on {ocp-short} by using the specified field in the Custom Resource.
```
Normally I'd say use custom resource (CR) and then CR for each instance after the first in a module, but I see this is used in other places, so more of a note for future updates :)